test/utils/cloud_provider_aws.go

```diff
@@ -130,6 +130,10 @@ func (p *awsVPC) GetVMNames() []string {
 	return vmNames
 }
 
+func (p *awsVPC) GetPrimaryNICs() []string {
```

Comment:
Is this required? Can't we re-use GetNICs() function

Reply:
No, Required for Azure case, comments have been added for pointing the difference between the 2 functions. GetPrimaryNICs() was required to be added to return the complete ID of NIC, GetNICs() returns just the name.